git.adoc

```diff
@@ -86,28 +86,28 @@ fi
 if [ $COMMIT_COUNT -gt 5 ]; then
   echo -e "Hmm...I'm counting $COMMIT_COUNT commits."
   echo "That seems a bit high."
-  echo "Check your pull request commit count in the GitHub PR. How many commits do you want to squash?"
-  read COMMIT_COUNT
+  echo "Check your pull request commit count in the GitHub PR."
+  read -p "How many commits do you want to squash? " COMMIT_COUNT
 fi
 
+#Check that a numerical value greater than zero was entered
+test "$COMMIT_COUNT" -gt 0
+
 #Prompt them on the commit count and branch name before commiting.
-echo "Do you want to squash $COMMIT_COUNT commits on branch $CURRENT_BRANCH to 1 commit?"
-echo "Answer Y/N followed by [Enter]: "
-read ANSWER
+read -p "Do you want to squash $COMMIT_COUNT commits on branch $CURRENT_BRANCH to 1 commit? [y/N] " ANSWER
 
 #Test to see if the user answers yes.
 if [ $ANSWER == "Y" -o $ANSWER = "y" ];
 then
   git reset --soft HEAD~$COMMIT_COUNT
   echo "Squashing $COMMIT_COUNT commits into 1 commit."
-  echo "Provide a commit message."
-  read COMMIT_MESSAGE
-  git commit -m "$CURRENT_BRANCH: $COMMIT_MESSAGE."
-  git push origin +$CURRENT_BRANCH
-else
-  echo "I didn't get a Y or y. Buh bye!"
-  exit
+  read -p "Provide a commit message: " COMMIT_MESSAGE 
+  git commit -m "$CURRENT_BRANCH: $COMMIT_MESSAGE"
+  git push --force-with-lease
```

Comment:
`--force-with-lease` fails if another user created a commit on the remote branch, and you would effectively be destroying their work.